src/dataconnect/client.ts

```diff
@@ -83,11 +83,12 @@ export async function deleteService(serviceName: string): Promise<types.Service>
 
 /** Schema methods */
 
-export async function getSchema(serviceName: string): Promise<types.Schema | undefined> {
+export async function getSchema(
+  serviceName: string,
+  schemaId: string,
```

Comment:
nit: we could give it a default `= types.SCHEMA_MAIN_ID` to reduce blast radius of this change.

Reply:
Done.